ui/src/app/sensors/components/sensor-list/sensor-list.tsx

```diff
@@ -0,0 +1,245 @@
+                            <div className='columns small-4'>NAME</div>
+                            <div className='columns small-3'>NAMESPACE</div>
+                            <div className='columns small-2'>CREATED</div>
+                            <div className='columns small-2'>LOGS</div>
+                        </div>
+                        {sensors.map(s => (
+                            <Link
+                                className='row argo-table-list__row'
+                                key={`${s.metadata.namespace}/${s.metadata.name}`}
+                                to={uiUrl(`sensors/${s.metadata.namespace}/${s.metadata.name}`)}>
+                                <div className='columns small-1'>
+                                    <i className='fa fa-clone' />
+                                </div>
+                                <div className='columns small-4'>{s.metadata.name}</div>
+                                <div className='columns small-3'>{s.metadata.namespace}</div>
+                                <div className='columns small-2'>
+                                    <Timestamp date={s.metadata.creationTimestamp} />
+                                </div>
+                                <div className='columns small-2'>
+                                    <div
+                                        onClick={e => {
+                                            e.preventDefault();
+                                            setSelectedNode(`${s.metadata.namespace}/Sensor/${s.metadata.name}`);
+                                        }}>
+                                        <i className='fa fa-bars' />
+                                    </div>
+                                </div>
+                            </Link>
+                        ))}
+                    </div>
+                </>
+            )}
+            <SlidingPanel isShown={sidePanel} onClose={() => setSidePanel(false)}>
+                <SensorCreator namespace={namespace} onCreate={s => navigation.goto(uiUrl(`sensors/${s.metadata.namespace}/${s.metadata.name}`))} />
+            </SlidingPanel>
+            <SlidingPanel isShown={!!selectedNode} onClose={() => setSelectedNode(null)}>
+                {!!selectedNode && (
+                    <div>
+                        <h4>
+                            Sensor/{selected.name}
+                            {selected.key ? '/' + selected.key : ''}
+                        </h4>
+                        <Tabs
+                            navTransparent={true}
+                            selectedTabKey={tab}
+                            onTabSelected={setTab}
+                            tabs={[
+                                {
+                                    title: 'LOGS',
+                                    key: 'logs',
+                                    content: (
+                                        <div>
```

Comment:
I think this should be a separate component

Reply:
👍